### DISCUSSION TITLE
请教一下，内核版本信息，编译的时候可以自定义吗？

### DISCUSSION THREAD

**Discussion by 2002yd** [ophub/amlogic-s9xxx-armbian#2559] [Amlogic]
各位大佬，请教一下，系统中uname -r显示的内核版本信息，例如：6.1.105-ophub，是否可以自定义？有查看了本地化打包相关说明，只看到设置内核版本这些选项，好像没有看到自定义内核版本信息的选项，麻烦帮忙看看，谢谢！

**Comment by ophub** [comment]
https://github.com/ophub/amlogic-s9xxx-armbian/tree/main/compile-kernel

**Reply by 2002yd** [reply]
感谢，查看了文档，CustomName参数设置，只能在内核后面增加自定义字符串，是否有参数，可以完全自定义内核版本信息呢，例如：6.1.105-ophub，我需要完全自定义，连前面的6.1.105都不要，谢谢！
